serde_macros/tests/compile-fail/duplicate_attributes.rs

```diff
@@ -0,0 +1,23 @@
+#![feature(custom_attribute, custom_derive, plugin)]
+#![plugin(serde_macros)]
+
+#[derive(Serialize)]
+struct S {
+    #[serde(rename(serialize="x"))]
+    #[serde(rename(serialize="y"))] //~ ERROR: duplicate serde attribute `rename`
```

Comment:
Awesome. Could you also add these cases?

* `#[serde(rename(serialize="x", serialize="x"))]`
*  `#[serde(rename="x", serialize="x"))]`
* ``#[serde(rename(serialize="x"), rename(serialize="x"))]`